### DISCUSSION TITLE
The stats are very different, can you tell me why?

### DISCUSSION THREAD

**Discussion by DongJu-Na** [anuraghazra/github-readme-stats#2785] [Q&A] [answered]
````
{
  "data": {
    "user": {
      "name": "DongJu-Na",
      "login": "DongJu-Na",
      "contributionsCollection": {
        "totalCommitContributions": 447,
        "restrictedContributionsCount": 3
      },
      "repositoriesContributedTo": {
        "totalCount": 12
      },
      "pullRequests": {
        "totalCount": 7
      },
      "openIssues": {
        "totalCount": 12
      },
      "closedIssues": {
        "totalCount": 19
      },
      "followers": {
        "totalCount": 5
      }
    }
  }
}
````

```
![GitHub Stat](https://github-readme-stats.vercel.app/api?username=DongJu-Na&show_icons=true&theme=neon&count_private=true)
```

![image](https://github.com/anuraghazra/github-readme-stats/assets/79893048/3564f6aa-6d5d-483a-8ada-6dceba9ec873)

Please let me know if there is anything I have set up incorrectly. Thank you.
:)

**Comment by qwerty541** [comment, marked as answer]
By default, the stats card only shows statistics like stars, commits and pull requests from public repositories. To show private statistics on the stats card, you should [deploy your own instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) using your own GitHub API token.

**Reply by DongJu-Na** [reply]
If I change the private storage to public, will the count be taken again?

**Reply by qwerty541** [reply]
> If I change the private storage to public, will the count be taken again?

Yes, if you change visability of your repository from private to public it will appear on your stats card. Remember that update can take some time due to caching, see #2764

**Reply by DongJu-Na** [reply]
thank you 😊
